packages/is-valid-titulo-eleitor/src/index.js

```diff
@@ -0,0 +1,26 @@
+import onlyNumbers from '@brazilian-utils/helper-only-numbers';
+
+import { BLACKLIST, TITULO_ELEITOR_LENGTH } from './constants';
+
+const isValidLength = tituloEleitor => tituloEleitor.length === TITULO_ELEITOR_LENGTH;
+
+const belongsToBlacklist = tituloEleitor => BLACKLIST.includes(tituloEleitor);
+
+const modSum = (arr, weight) => arr.reduce((acc, digit) => weight < 10 ? acc + digit * weight++ : 0, 0) % 11;
+
+const isValid = (arr, mod1, mod2) => (arr[10] === mod1 && arr[11] === mod2) && ((arr[8] + arr[9] > 0) && (arr[8] + arr[9] < 29));
+
+const isValidChecksum = tituloEleitor => {
+    const digits = Array.from(tituloEleitor).map(Number)
+
+    return isValid(digits, modSum(digits.slice(0,8), 2), modSum(digits.slice(8,11), 7));
```

Comment:
I didn't understand the magic numbers in this function :( I think you should describe with variables what they are.